spatialprofilingtoolbox/apiserver/app/main.py

```diff
@@ -375,3 +378,60 @@ async def get_plot_high_resolution(
     def streaming_iteration():
         yield from input_buffer
     return StreamingResponse(streaming_iteration(), media_type="image/png")
+
+
+@app.get("/importance-fraction-plot/")
+async def importance_fraction_plot(
+    study: ValidStudy,
+    img_format: str = 'svg',
+) -> StreamingResponse:
+    """Return a plot of the fraction of important cells expressing a given phenotype."""
+    APPROVED_FORMATS = {'png', 'svg'}
+    if img_format not in APPROVED_FORMATS:
+        raise ValueError(f'Image format "{img_format}" not supported.')
```

Comment:
I think this validation might be better implemented in the same system as everything else (FastAPI's recommendation). So validation failure is handled more gracefully by FastAPI.